module/applications/advancement/advancement-manager.mjs

```diff
@@ -303,6 +381,17 @@ export default class AdvancementManager extends Application {
       .map(a => new a.constructor.metadata.apps.flow(item, a.id, level));
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Determine the proper working level either from the provided item or from the cloned actor.
+   * @param {Item5e} item  Item being advanced. If class or subclass, its level will be used.
+   * @returns {number}     Working level.
+   */
+  currentLevel(item) {
+    return item.system.levels ?? item.class?.system.levels ?? manager.clone.system.details.level;
+  }
```

Comment:
I guess this also needs to take an actor parameter since `manager` is not defined in this context:

```suggestion
  currentLevel(item, actor) {
    return item.system.levels ?? item.class?.system.levels ?? actor.system.details.level;
  }
```